doc/documentation_data_format.md

```diff
@@ -366,9 +366,42 @@ Additional columns may be added.
     - parameterScaleLaplace: location; scale
 
 
-## Parameter estimation problems combining multiple models
+### Additional optional columns
 
-[**Issue #49**](https://github.com/ICB-DCM/PEtab/issues/49)
+Extra columns:
+
+- `hierarchicalOptimization` (optional)
+
+  hierarchicalOptimization: 1 if parameter is optimized using hierarchical
```

Comment:
not completely sure yet if this will be applicable, or if we need more info (e.g. on whether it's an offset, scaling, sigma etc.; this info should be possible to extract from the mathml, but there may be special cases).

Reply:
a pypesto implementation is in a preliminary state, for the moment I think we can just leave this here as is, since it's optional anyway.

Reply:
This was old, just moved two blocks. I think those extensions need some more explanation. For hierarchicalOptimization it is moderately unclear what happens. I think would remove that and put it rather into pyPESTO as "PEtab extensions". Maybe not in this PR though.